app/SearchApi/BcGov.Fams3.SearchApi.Contracts/Person/Phone.cs

```diff
@@ -12,6 +12,6 @@ public class Phone : PersonalInfo
         public string Type { get; set; }
 
         [Description("The owner of the phone e.g. Applicant, Person Sought, e.t.c")]
-        public OwnerType Owner { get; set; }
+        public OwnerType Owner { get; set; } = OwnerType.NotApplicable;
     }
```

Comment:
We shouldn't put this in the contract. It should be in the mapping or separate definition